akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -412,7 +438,19 @@ TODO
 Fine-tuning CPU usage latency tradeoff
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-TODO
+Artery has been designed for low latency and as a result it can be CPU hungry when the system is mostly idle.
+This is not always desirable. It is possible to tune the tradeoff between CPU usage and latency with
+the following configuration:
+
+  # Values can be from 1 to 10, where 10 strongly prefers latency
+  # and 1 strongly prefers less CPU usage
+  akka.remote.artery.advanced.idle-cpu-level = 1
+
+By setting this value to a lower number, it tells Akka to do longer "sleeping" periods on its thread dedicated
+for `spin-waiting <https://en.wikipedia.org/wiki/Busy_waiting>`_ and hence reducing CPU load when there is no
+immediate task to execute at the cost of a longer reaction time to an event when it actually happens. It is worth
+to be noted though that during a continuously high-throughput period this setting makes not much difference
+as the thread mostly has tasks to execute.
 
```

Comment:
Perhaps add something like:

This behavior is also the reason why you will see lower latency at a high message rate (10,000 messages/s) than at a low message rate (100 messages/s) as long as the system is not reaching saturation.